charts/prometheus-kafka-exporter/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "v1.3.1"
+appVersion: "v1.3.2"
 description: A Helm chart to export the metrics from Kafka in Prometheus format using the kafka-exporter from https://github.com/danielqsj/kafka_exporter
 name: prometheus-kafka-exporter
 home: https://github.com/danielqsj/kafka_exporter
-version: 1.4.0
+version: 1.4.1
```

Comment:
Adding such functionality, we should go to 1.5.0, I think

Reply:
i agree